### ISSUE TITLE
Unable to start server

### ISSUE THREAD

**Issue by omotayoR1** [callstack/repack#548]
I upgraded re.pack to @4.0.0-rc.0 and i'm not able to start the server by running 'npx react-native webpack-start' again. The error i'm getting below;

error fastify-favicon: \C:\Users\directory\Documents\d\node_modules@callstack\repack-dev-server\dist\img\favicon.ico not found.
Error: fastify-favicon: \C:\Users\directory\Documents\d\node_modules@callstack\repack-dev-server\dist\img\favicon.ico not found
at ReadFileContext. (C:\Users\directory\Documents\d\node_modules\fastify-favicon\src\plugin.js:60:14)
at ReadFileContext.callback (C:\Users\directory\Documents\d\node_modules\graceful-fs\graceful-fs.js:123:16)
at FSReqCallback.readFileAfterOpen [as oncomplete] (node:fs:306:13)

what do i do

Thanks for helping

**Comment by omotayoR1** [comment]
> @omotayoR1 I have no access to Windows right now and have no way to check for a fix.
> 
> My best guess is that somehow path in `C:\Users\directory\Documents\d\node_modules@callstack\repack-dev-server\dist\plugins\faviconPlugin.js` evaluates incorrectly, you can try adjust it to see if it resolves this.

I checked the path mentioned and on line 24, it seems something is wrong there const fp = require('fastify-plugin') 

What's the resolution please

Thanks for helping

**Comment by jbroma** [comment]
@omotayoR1 could you provide a snippet? I see the file has only like 15 LOC

**Comment by jbroma** [comment]
@omotayoR1 FYI, I should be able to investigate this on Windows by the end of the week

**Comment by omotayoR1** [comment]
> @omotayoR1 FYI, I should be able to investigate this on Windows by the end of the week

Okay

Thanks for helping

**Comment by paaber** [comment]
@jbroma  have you been able to iveestigate this ?


> @omotayoR1 FYI, I should be able to investigate this on Windows by the end of the week

**Comment by jbroma** [comment]
@paaber yes, the investigation is ongoing and we're working on a fix together with @szymonrybczak

**Comment by jbroma** [comment]
@omotayoR1 @paaber we've just released `@callstack/repack` 4.0.0-rc.1, your issues should be resolved now :)

Btw, you can now install Re.Pack in new projects with custom version like this: 
```
npx @callstack/repack-init@latest --custom-version 4.0.0-rc.1
```

**Comment by omotayoR1** [comment]
> @omotayoR1 @paaber we've just released `@callstack/repack` 4.0.0-rc.1, your issues should be resolved now :)
> 
> Btw, you can now install Re.Pack in new projects with custom version like this:
> 
> ```
> npx @callstack/repack-init@rc --custom-version 4.0.0-rc.1
> ```

Thanks for helping. The server starts now and builds successfully but it stops bundling at some point and it throws this error
Error [ERR_UNHANDLED_ERROR]: Unhandled error. ({})
    at Compiler.emit (node:events:508:17)
    at Worker.<anonymous> (C:\Users\directory\Documents\anotherHastro\node_modules\@callstack\repack\dist\webpack\Compiler.js:97:14)
    at Worker.emit (node:events:519:28)
    at MessagePort.<anonymous> (node:internal/worker:263:53)
    at [nodejs.internal.kHybridDispatch] (node:internal/event_target:822:20)
    at exports.emitMessage (node:internal/per_context/messageport:23:28) {
  code: 'ERR_UNHANDLED_ERROR',
  context: {}
}

Node.js v21.4.0

Please help me look into it

Thanks

**Comment by jbroma** [comment]
@omotayoR1 please open a new issue
